imap_processing/quality_flags.py

```diff
@@ -66,3 +66,26 @@ class HitFlags(
     INF = CommonFlags.INF  # bit 0
     NEG = CommonFlags.NEG  # bit 1
     FLAG3 = 2**2  # bit 2
+
+
+class SWAPIFlags(
+    FlagNameMixin,
+):
+    """SWAPI flags."""
+
+    NONE = CommonFlags.NONE
+    INF = CommonFlags.INF  # bit 0
+    NEG = CommonFlags.NEG  # bit 1
+    SWP_PCEM_COMP = 2**2  # bit 2
```

Comment:
We might add the ability to have comments for each Flag. But for now could you put in a short docstring comment for each of these? 

Another question: are the bad data indicators for the sweeps included here?

   1. SWP_HK.CHKSUM is wrong
   2. SWAPI mode (SWP_SCI.MODE) is not HVSCI
   3. PLAN_ID for current sweep should all be one value
   4. SWEEP_TABLE should all be one value.

Reply:
Good point. I believe these bad data indicators won't be stored as flag. They are used to filter bad data before it get saved. Current flags are then set on good data only for L3 purposes.